okhttp-ws/src/main/java/com/squareup/okhttp/internal/ws/WebSocketProtocol.java

```diff
@@ -71,11 +71,13 @@
    * Maximum length of frame payload. Larger payloads, if supported, can use the special values
```

Comment:
This says _if supported_ . . . below it looks like we're assuming these will always be supported?